src/plots/gl3d/camera.js

```diff
@@ -48,6 +48,7 @@ function createCamera(element, options) {
 
     var camera = {
         keyBindingMode: 'rotate',
+        enableWheel: true,
```

Comment:
Could we use `disableWheel` instead?

Reply:
I would prefer keeping this `enableWheel` so that on this line

https://github.com/plotly/plotly.js/blob/16a01e7d4fd388f68b3bce61af0a362e6770d300/src/plots/gl3d/scene.js#L421

it matches `gd._context._scrollZoom.gl3d`.